.github/workflows/ubuntu.yml

```diff
@@ -1,37 +1,76 @@
 name: Celix Ubuntu
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '0 0 * * *'
 
 jobs:
-  build:
-    runs-on: ${{ matrix.os }}
+
+  build-conan:
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        #os: [ubuntu-20.04]
-        #compiler: [gcc, clang]
-        include:
-          - os: ubuntu-20.04
-            name: clang
-            sanitizer: false #note sanitizer on clang with cpputest does not work
-            compiler: clang
-            cxx_compiler: clang++
-          - os: ubuntu-20.04
-            name: gcc
-            sanitizer: true
-            compiler: gcc
-            cxx_compiler: g++
+        compiler: [ [gcc,g++], [clang,clang++] ]
+        type: [ Debug ]
+    timeout-minutes: 120
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3.3.0
+      - name: Install conan
+        run: |
+          sudo pip install conan
```

Comment:
When using conan-center-index, we have to use the latest conan client. Otherwise it will stop working after some upstream upgrade.